tests/unit/test_package.py

```diff
@@ -1805,3 +1806,83 @@ def test_default_service_config_to_text(self):
         self.assertTrue(
             compare_xml(app_package.services_to_text_vt, expected_result),
         )
+
+    def test_document_expiry(self):
+        # Create a Schema with name music and a field with name artist, title and timestamp
+        # Ref https://docs.vespa.ai/en/documents.html#document-expiry
+        application_name = "music"
+        music_schema = Schema(
+            name=application_name,
+            document=Document(
+                fields=[
+                    Field(
+                        name="artist",
+                        type="string",
+                        indexing=["attribute", "summary"],
+                    ),
+                    Field(
+                        name="title",
+                        type="string",
+                        indexing=["attribute", "summary"],
+                    ),
+                    Field(
+                        name="timestamp",
+                        type="long",
+                        indexing=["attribute", "summary"],
+                        attribute=["fast-access"],
+                    ),
+                ]
+            ),
+        )
+        # Create a ServicesConfiguration with document-expiry set to 1 day (timestamp > now() - 86400)
+        services_config = ServicesConfiguration(
+            application_name=application_name,
+            services_config=services(
```

Comment:
Starting here linters complain that services, container, etc are not defined.
They can't see dynamically generated imports.

Reply:
Ref offline chat, I consider this as resolved for now, and taking a look on whether we can support this in the future. 
When using linter-settings as defined in`pyproject.toml` these errors are ignored.